kt/godot-library/src/main/kotlin/godot/core/Functions.kt

```diff
@@ -17,8 +17,10 @@ enum class PropertyHint {
     FLAGS, ///< hint_text= "flag1,flag2,etc" (as bit flags)
     LAYERS_2D_RENDER,
     LAYERS_2D_PHYSICS,
+    LAYERS_2D_NAVIGATION,
```

Comment:
Before we had to copy multiple enums in the godot library because the generated code was in a different module but now that everything is together, wouldn't that be better to directly use the one available in gen/ ?

Reply:
I agree. As discussed on Discord I will look into this and file a separate PR.